crates/biome_cli/src/commands/mod.rs

```diff
@@ -151,6 +152,12 @@ pub enum BiomeCommand {
 
         #[bpaf(external, hide_usage)]
         cli_options: CliOptions,
+
+        /// Run only the given rule taking into account the options set in the configurations files.
+        /// The severity level of the rule is set to its default.
+        #[bpaf(long("rule"), argument("NAME"))]
```

Comment:
- we could add two examples, one for executing a rule, one for executing a group. This will appear in the help section and reference page
- instead of `NAME` we could have `GROUP|RULE_NAME` (not that great as a suggestion though, feel free to discard it)